components/notebook-controller/config/overlays/openshift/kustomization.yaml

```diff
@@ -8,17 +8,27 @@ commonLabels:
   app.kubernetes.io/part-of: odh-notebook-controller
```

Comment:
downstream does not have `namespace:` on line 6, is that necessary to have it hardcoded here, or should the operator be responsible for deciding the namespace depending on user configuration?

Reply:
Hmm not sure, probably there is some reason @harshad16 wdyt? 
Upstream: https://github.com/opendatahub-io/kubeflow/blob/v1.7-branch/components/notebook-controller/config/overlays/openshift/kustomization.yaml#L6 
Downstream: https://github.com/red-hat-data-services/kubeflow/blob/master/components/notebook-controller/config/overlays/openshift/kustomization.yaml

Reply:
i m not aware, let ask this to operator team.
for now, lets keep this as it is.